spec/support/dummy_server/servlet.rb

```diff
@@ -172,5 +172,20 @@ def do_#{method.upcase}(req, res)
                    "#{req.body}-raw"
                  end
     end
+
+    post "/no-content-204" do |req, res|
+      res.status = 204
+
+      res.body = case req["Accept-Encoding"]
+                 when "gzip" then
+                   res["Content-Encoding"] = "gzip"
+                   ""
+                 when "deflate" then
+                   res["Content-Encoding"] = "deflate"
+                   ""
+                 else
+                   ""
+                 end
```

Comment:
RuboCop suggests moving the `""` out of the conditional since it's the same for all branches

```suggestion
      res.body = ""
      case req["Accept-Encoding"]
      when "gzip" then
         res["Content-Encoding"] = "gzip"
       when "deflate" then
         res["Content-Encoding"] = "deflate"
        end
```

Reply:
@tarcieri fair suggestions, fixed.